zyte_common_items/items.py

```diff
@@ -629,7 +642,7 @@ class ProductFromList(Item):
     mainImage: Optional[Image] = None
 
     #: Data extraction process metadata.
-    metadata: Optional[Metadata] = None
+    metadata: Optional[ProbabilityMetadata] = None
```

Comment:
I think it'd be nice to have a converter here as well, so that passing Metadata works everywhere.